docs/guide/static-deploy.md

```diff
@@ -62,42 +62,57 @@ Now the `preview` command will launch the server at `http://localhost:8080`.
 
    If you are deploying to `https://<USERNAME>.github.io/<REPO>/`, for example your repository is at `https://github.com/<USERNAME>/<REPO>`, then set `base` to `'/<REPO>/'`.
 
-2. Inside your project, create `deploy.sh` with the following content (with highlighted lines uncommented appropriately), and run it to deploy:
-
-   ```bash{16,24,27}
-   #!/usr/bin/env sh
-
-   # abort on errors
-   set -e
-
-   # build
-   npm run build
-
-   # navigate into the build output directory
-   cd dist
-
-   # place .nojekyll to bypass Jekyll processing
-   echo > .nojekyll
-
-   # if you are deploying to a custom domain
-   # echo 'www.example.com' > CNAME
-
-   git init
-   git checkout -B main
-   git add -A
-   git commit -m 'deploy'
-
-   # if you are deploying to https://<USERNAME>.github.io
-   # git push -f git@github.com:<USERNAME>/<USERNAME>.github.io.git main
-
-   # if you are deploying to https://<USERNAME>.github.io/<REPO>
-   # git push -f git@github.com:<USERNAME>/<REPO>.git main:gh-pages
-
-   cd -
+2. Go to your GitHub Pages configuration and choose the source of deployment as "GitHub Actions", this will lead you to create a workflow that builds and deploys your project, a sample workflow that installs dependencies and builds using Yarn is provided:
+
+   ```yml
+   # Simple workflow for deploying static content to GitHub Pages
+   name: Deploy static content to Pages
+
+   on:
+     # Runs on pushes targeting the default branch
+     push:
+       branches: ['master']
```

Comment:
Correct, I based this workflow in one of the default configurations on GitHub, but it seems it changes the workflow based on your main branch, mine was still 'master' because it was created before this change was incorporated.